packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -635,3 +641,5 @@ export interface IDetailsGroupRenderProps extends IGroupRenderProps {
 }
 
 export interface IDetailsGroupDividerProps extends IGroupDividerProps, IDetailsItemProps {}
+
+export interface IDetailsListCheckboxProps extends IDetailsCheckboxProps {}
```

Comment:
If we don't need to redefine an interface I'd just export it. But I'd like to see if we can leverage `Pick<T, [key of T]>` as mentioned in my comment below.